lib/hyrax/specs/shared_specs/hydra_works.rb

```diff
@@ -26,6 +26,63 @@
   it { is_expected.to respond_to :work? }
 end
 
+RSpec.shared_examples 'belongs to collections' do
+  subject(:model)        { described_class.new }
+  let(:adapter)          { Valkyrie::MetadataAdapter.find(:test_adapter) }
+  let(:persister)        { adapter.persister }
+  let(:query_service)    { adapter.query_service }
+  let(:collection_class) { Hyrax::PcdmCollection }
+
+  describe 'collection membership' do
+    it 'is in no collections default' do
```

Comment:
The wording of this doesn't quite read right. Do you mean `it 'is in no collections by default'`?